google/cloud/bigtable/examples/bigtable_table_admin_backup_snippets.cc

```diff
@@ -207,17 +278,19 @@ void SetIamPolicy(google::cloud::bigtable::TableAdmin const& admin,
       *current->add_bindings() = cbt::IamBinding(role, {member});
     }
     StatusOr<google::iam::v1::Policy> policy =
-        admin.SetIamPolicy(cluster_id, backup_id, *current);
+        admin.SetIamPolicy(backup_name, *current);
     if (!policy) throw std::runtime_error(policy.status().message());
     std::cout << "The IAM Policy is:\n" << policy->DebugString() << "\n";
   }
   //! [set backup iam policy]
-  (std::move(admin), argv.at(0), argv.at(1), argv.at(2), argv.at(3));
+  (std::move(admin), argv.at(0), argv.at(1), argv.at(2), argv.at(3), argv.at(4),
+   argv.at(5));
 }
 
 void RunAll(std::vector<std::string> const& argv) {
   namespace examples = ::google::cloud::bigtable::examples;
   namespace cbt = ::google::cloud::bigtable;
+  namespace cbta = ::google::cloud::bigtable_admin;
 
   if (!argv.empty()) throw examples::Usage{"auto"};
   if (!examples::RunAdminIntegrationTests()) return;
```

Comment:
Note that these examples are not executed on each PR, you may want to run them manually to make sure they work.

Reply:
can confirm that the `RunAll` example works :+1: